client/components/data/query-sites/README.md

```diff
@@ -0,0 +1,8 @@
+Query Sites
+===========================
+
+`<QuerySites />` is a React component used in managing network requests for sites.
```

Comment:
I think we should clarify in the readme that this will trigger a vanilla request against the `/me/sites` endpoint, to make it clear it will not for example hit the `/sites/:site` endpoint. I wonder if we should even change the name to `QueryMeSites`.